### ISSUE TITLE
Jaws screenreader doesn't read the braces

### ISSUE THREAD

**Issue by gFrincu** [microsoft/vscode#204610]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.86.0
- OS Version: windows 10/11, jaws 2024 and nvda 2023.3

Steps to Reproduce:

1. Write code containing braces like `export const Navbar = () => {}`.
2. Navigate through the code using the arrow keys with JAWS 2024 enabled.
3. Observe that JAWS does not read the braces as expected, it reads blank like they didn't exist.
4. Do insert shift 2 and change the punctuation jaws is reading to all the punctuation.
5. Check that jaws still doesn't read the braces.

This issue seems to have emerged in the latest version of VS Code, as previous versions worked fine with JAWS. Currently, NVDA appears to handle the code navigation correctly.

**Comment by gFrincu** [comment]
I didn't see that this one existed, closing issue.
https://github.com/microsoft/vscode/issues/200956
